src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -142,6 +142,21 @@
 				:icon="mdiChevronRight"
 			/>
 		</kv-button>
+
+		<!-- If allSharesReserved show message and hide cta button -->
+		<div
+			v-if="allSharesReserved"
+			class="
+				tw-rounded
+				tw-bg-brand-100
```

Comment:
All good for this PR, we can tweak this after talking to design, but we'll want to remove `tw-bg-brand-100` in favor of a themable background color in the near-future.